### ISSUE TITLE
Still alive?

### ISSUE THREAD

**Issue by randallb** [mauricemach/coffeekup#110]
@mauricemach, you still planning to update CoffeeKup? There are like 10 pull requests pending, and I'm just wondering if you're going to keep maintaining this.

**Comment by randallb** [comment]
As a sub aside, with Express 3, they remove support for Layouts and make the template engine the one to deal with them. I figure that'll mean some changes to docs, at the bare minimum. I'm willing to update docs / howtos if you're planning to keep things going.

**Comment by ejsmith** [comment]
Apparently not.  What a shame.

**Comment by randallb** [comment]
Someone who's interested in maintaining it should fork it and start maintaining it. I'm moving to Eco, most likely.

**Comment by jussiry** [comment]
This is such a shame, I really hope someone take up this great project. And @mauricemach, please let us know if you're not interested in maintaining it anymore.

**Comment by gradus** [comment]
Not sure I have the time to maintain properly, but have forked the project and will work on resolving issues and merging in existing pull request from the original project into my fork.  This is only for my own use b/c I really like coffeekup.  If anyone is interested in helping let me know.  May be a week or so before I have time to work on any of it.

**Comment by n1k0** [comment]
I'm getting worry about Maurice. He doesn't answer to emails, twitter, nothing. Anybody having actual news about him?

**Comment by randallb** [comment]
I don't know him personally... just used coffeekup.

**Comment by gradus** [comment]
I am going to keep using and maintaining this fork https://github.com/gradus/coffeekup  Have already merged in a few of the pull requests.  Any help on getting existing issues into this fork and helping with pull requests would be appreciated.  I don't mind taking over this project at all.  just need a little help.  this is a great project and I want to keep it alive.  anyone interested in helping?

**Comment by gradus** [comment]
@mauricemach hope you are not in jail or having some sort of difficulty that has prevented you from communicating.  I would like to take over this project if you are no longer interested or are unable to do so.

**Comment by randallb** [comment]
I'm interested in helping. I've never really contributed or led an open source project before, but I'd love to help cofound it. @gradus hit me up with a gh message if you wanna talk about how to keep things going, your priority, etc.

**Comment by gradus** [comment]
@randalb my priority at the moment is to review the existing pull requests that I feel need to be merged into my fork, add the code manually and then add all of the current issues to this github fork ( https://github.com/gradus/coffeekup ) and start tackling them one by one. Adding issues from @mauricemach / coffekup to my fork is a good way to start.  Then, we can go from there. Everyone should be contributing to open source.  Glad to see your interested.  I am just using coffeekup in a few of my NodeJS projects and the idea of coffeescript templates really appeals to me. I have to keep this project alive and am willing to take as much time as needed to do so. We may want to rename the project at some point before packaging and releasing npm module.  Would love to here @mauricemach 's take on this idea, but he seems to be MIA.

**Comment by ejsmith** [comment]
Awesome!  Glad you guys are interested in keeping this project alive.  It seems like an awesome templating system.  I would love to see it being improved and I will try to help any way I can.  I am still new to this entire world though and I'm slowly getting up to speed.

**Comment by gradus** [comment]
I have renamed the project and have moved it here https://github.com/gradus/coffeecup.  Will continue to keep it alive as a clone of the original coffeekup.  thanks @mauricemach

**Comment by gradus** [comment]
Just published npm module coffeecup.  was able to simply require('coffeecup') in existing node v0.6.11 project and seems to work great.  https://github.com/gradus/coffeecup is my new clone.  contributions welcome.  please help me keep this project alive.

**Comment by randallb** [comment]
I just added a bunch of issues that I thought were valid, etc. I'm not sure the best way to deal with this repo since it's kind of a goner now.

**Comment by gradus** [comment]
Much appreciated @randallb.

**Comment by ghost** [comment]
thanks guys for keeping coffeekup alive. will move to coffeecup now. thanks!

**Comment by gradus** [comment]
@marvin Always open to contributions.  Working on fixing a few issues and adding a few new features at the moment.  Please submit an issue if you run across any problems. I have put up a couple boilerplate examples on my github repo using the new CoffeeCup with flatiron and tako web frameworks (black-coffee and macho-tako).  Documentation is now at http://easydoc.org/coffecup.

**Comment by ghost** [comment]
haha just put up a conversion of the html5 boilerplate to coffeecup :-)
thanks for the link.

**Comment by twilson63** [comment]
if anyone is interested.

I just added some CoffeeCup template handlers for flatiron web framework http://flatironjs.org

http://github.com/twilson63/creamer
http://github.com/twilson63/flatiron-handlers

**Comment by wmertens** [comment]
@gradus Perhaps you can respond to all open issues pointing out this issue and proclaiming yourself the new maintainer?

If @mauricemach stays MIA for much longer perhaps the github staff can be persuaded to give you access to this repository...

Likewise perhaps the NPM registry could point to your clone? Coffeecup is not very google-able and you would have to know to search for it in the first place :-/

**Comment by ghost** [comment]
@gradus if you want assign me anything you want and I'll take care of it. FYI

**Comment by gradus** [comment]
@wmertens not a bad idea as far as responding to all open issues.  I have discussed with the only other collaborator at this point, @twilson63, the difficulty of finding the project without the 'K".  I obviously made the name change choice b/c of the ability to push to npm.  I think we are going to leave the name CoffeeCup for now and do our best to get the word out as it gains a little more traction.  Your idea about having the npm registry point to the clone is feasible and I will look into it.

**Comment by aseemk** [comment]
This is actually quite worrying that @mauricemach is MIA. =/ I hope he's okay...

**Comment by gradus** [comment]
I hope it is a disappearance that was of his own choosing.
This project was based on Markaby.
_why disappeared.

now Maurice after this project.

Hoping I don't happen to disappear next.

**Comment by aseemk** [comment]
Indeed, I've just noticed that. Thanks @gradus. I hope that this was his own choosing too and that he's okay.

**Comment by gradus** [comment]
@marvin you still interested in helping.  I know we have made the decision to minimize adding any new features but as @randallb pointed out, 'Error reporting has much to be desired'.  The following issue will be my next top priority, if you are interested in contributing.  https://github.com/gradus/coffeecup/issues/10

**Comment by ghost** [comment]
@gradus of course I am was just a bit busy with some private stuff and eastern etc 
I'll take a look tonight cause have to work right now :-)

**Comment by auvipy** [comment]
#121
